packages/webcomponents/src/components/payments-list/payments-list-core.tsx

```diff
@@ -93,53 +149,58 @@ export class PaymentsListCore {
             />
           </div>
         </div>
-        <justifi-table
-          rowClickHandler={e => {
-            const clickedPaymentID = e.target.closest('tr').dataset.rowEntityId;
-            if (!clickedPaymentID) { return }
-            this.rowClicked.emit(this.payments.find((payment) => payment.id === clickedPaymentID));
-          }}
-          entityId={this.payments.map((payment) => payment.id)}
-          columnData={[
-            ['Made On', 'The date and time each payment was made'],
-            ['Amount', 'The dollar amount of each payment'],
-            ['Description', 'The payment description, if you provided one'],
-            ['Cardholder', 'The name associated with the payment method'],
-            ['Payment Method', 'The brand and last 4 digits of the payment method'],
-            ['Status', 'The current status of each payment'],
-            ['Payment ID', 'The unique identifier of each payment']
-          ]}
-          rowData={
-            this.payments.map((payment) => (
-              [
-                {
-                  type: 'head',
-                  value: `
-                    <div class='fw-bold'>${formatDate(payment.created_at)}</div>
-                    <div class='fw-bold'>${formatTime(payment.created_at)}</div>
-                  `,
-                },
-                formatCurrency(payment.amount),
-                payment.description,
-                payment.payment_method.payersName,
-                payment.payment_method.lastFourDigits,
-                {
-                  type: 'inner',
-                  value: MapPaymentStatusToBadge(payment.status)
-                },
-                payment.id
-              ]
-            ))
-          }
-          loading={this.loading}
-          error-message={this.errorMessage}
-          params={this.params}
-          paging={{
-            ...this.paging,
-            handleClickNext: this.handleClickNext,
-            handleClickPrevious: this.handleClickPrevious
-          }}
-        />
+        <div class="table-wrapper">
+          <table class="table table-hover">
+            <thead class="table-head sticky-top" part="table-head">
+              <tr class="table-light text-nowrap" part="table-head-row">
+                {this.columnData?.map((column) => (
+                  <th part="table-head-cell" scope="col" title={Array.isArray(column) ? column[1] : ''}>
+                    {!Array.isArray(column) ? column : column[0]}
+                  </th>
+                ))}
+              </tr>
+            </thead>
+            <tbody class="table-body" part="table-body">
+              {this.loading && LoadingState(this.columnData.length)}
+              {this.showEmptyState && EmptyState(this.columnData.length)}
+              {this.showErrorState && ErrorState(this.columnData.length, this.errorMessage)}
```

Comment:
I have a small suggestion, but feel free to ignore if you want, or perhaps save it for another PR if you agree but dont want to add to this PR. 

What if we updated these TableState functions / logics to FunctionalComponents? 

I think it could be nicer to read a few component tags here instead of the boolean logic, and then maybe you could move the render logic inside the function component (this.showEmptyState, this.showErrorState, etc)

```
export const TableErrorState = (props: TableRenderStateProps) => {
  const { columnSpan, errorMessage } = props;

  if (!columnSpan || !errorMessage) {
    return null;
  }

  return (
    <tr>
      <td class="error-state" part="error-state" colSpan={columnSpan} data-test-id="table-error-state" style={{ textAlign: 'center' }}>
        An unexpected error occurred: {errorMessage}
      </td>
    </tr>
  );
};
```

and then could be rendered in the list component as such. 🤔 

```
<tbody class="table-body" part="table-body">
      <TableLoadingState columnSpan={this.columnData.length} />
      <TableEmptyState columnSpan={this.columnData.length} />
       <TableErrorState columnSpan={this.columnData.length} errorMessage={this.errorMessage} />
........
```

No pressure either way, just might be nice eventually

Reply:
Nice suggestion!